packages/main/src/types/BadgeDesign.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Defines badge design types.
+ *
+ * @readonly
+ * @enum {string}
+ * @public
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.BadgeDesign
+ */
+enum BadgeDesign {
+	/**
+	 * Set1 of generic indication colors
+	 * @public
+	 * @type {Set1}
+	 */
+	Set1 = "Set1",
+
+	/**
+	 * Set2 of generic indication colors
+	 * @public
+	 * @type {Set2}
+	 */
+	Set2 = "Set2",
+
+	/**
+	 * Set3 of generic indication colors used in BTP design system
```

Comment:
Set3 of generic indication colors that are intended only for industry-specific use cases. Used in SAP BTP design system.